PR: https://github.com/odoo/odoo/pull/112224

From: 6fac5173996a28988bed6353889d3e72793de80e
From: Fernanda Hernández
Date: 2023-02-09 08:47:04

Structural Changes: 3
Total Changes: 6

[REF] im_livechat: make translatable some fields in im_livechat.channel model

Fields are:

1. button_text
2. default_message
3. input_placeholde

closes odoo/odoo#112224

Signed-off-by: Sébastien Theys (seb) <seb@odoo.com>

================================= pseudo patch: =================================

--- a/addons/im_livechat/models/im_livechat_channel.py
+++ b/addons/im_livechat/models/im_livechat_channel.py
@@ -29,10 +29,10 @@ class ImLivechatChannel(models.Model):
     # attribute fields
     name = fields.Char('Channel Name', required=True)
     button_text = fields.Char('Text of the Button', default=_('Have a Question? Chat with us.'),
-        help="Default text displayed on the Livechat Support Button")
+        help="Default text displayed on the Livechat Support Button", translate=True)
     default_message = fields.Char('Welcome Message', default='How may I help you?',
-        help="This is an automated 'welcome' message that your visitor will see when they initiate a new conversation.")
-    input_placeholder = fields.Char('Chat Input Placeholder', help='Text that prompts the user to initiate the chat.')
+        help="This is an automated 'welcome' message that your visitor will see when they initiate a new conversation.", translate=True)
+    input_placeholder = fields.Char('Chat Input Placeholder', help='Text that prompts the user to initiate the chat.', translate=True)
     header_background_color = fields.Char(default="#875A7B", help="Default background color of the channel header once open")
     title_color = fields.Char(default="#FFFFFF", help="Default title color of the channel once open")
     button_background_color = fields.Char(default="#875A7B", help="Default background color of the Livechat button")
